config/database.yml

```diff
@@ -68,8 +68,8 @@ test:
   database: capstone_back_end_test
   #####################
   # Omar configuration
-  # username: postgres
-  # password: 1234
+  username: postgres
+  password: 1234
```

Comment:
- [ ] [OPTIONAL] Great work @BenjaminSemah, It will be better next time when you make a PR to make sure that you comment your credentials to prevent conflict with other partners.

Reply:
Well noted @omarsalem7 
Thank you.